### PR TITLE
Webpack: remove TinyMCE external dependency mapping

### PR DESCRIPTION
**Pull request by ellatrix**
## Description

This is no longer used. Additionally removes an old irrelevant comment.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate.

### CODE REVIEW

**Review by gziolo — APPROVED** [review]
Looks good, thanks for cleaning it up 👍

File: webpack.config.js [hunk 1]
```diff
@@ -48,7 +48,6 @@ const gutenbergPackages = Object.keys( dependencies )
 const externals = {
 	react: 'React',
 	'react-dom': 'ReactDOM',
-	tinymce: 'tinymce',
```
Comment (youknowriad) [inline webpack.config.js]: Don't we use this in the classic editor?

Comment (iseulde) [reply webpack.config.js]: No, it directly uses the `window` global.

Comment (youknowriad) [reply webpack.config.js]: Ok that's fine for me.

**Comment by iseulde** [comment]
Thanks!

**Pull request merged**